components/Footer.tsx

```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+import { FC } from "react";
+
+const Footer: FC = () => {
+  return (
+    <div className="mx-auto mb-10 flex w-screen justify-center">
+      <a href="https://vercel.com/avneesh0612?utm_source=buidlers-hub&utm_campaign=oss">
```

Comment:
I guess it should be `https://vercel.com/?utm_source=buidlers-hub&utm_campaign=oss`